### PR TITLE
JI:765256 P2022-1249 1. Setup basic store with sample reducer

### PR DESCRIPTION
**Pull request by krysia1**

### CODE REVIEW

File: libs/shared/data/feedback/mocks/sample-reducer/selectors.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const selectSampleText = (state: { sample: { text: string } }) => state.sample.text;
```
Comment (kacperChlon) [inline libs/shared/data/feedback/mocks/sample-reducer/selectors.ts:1]: I would avoid destructuring here, sometimes you don't know if you've got the value, moreover what's the point in destructuring if you are using state.sample.text? ;)

File: libs/shared/data/feedback/mocks/sample-reducer/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export * from "./actions";
```
Comment (kacperChlon) [inline libs/shared/data/feedback/mocks/sample-reducer/index.ts:1]: Why those files are in mocks directory? :)

Comment (krysia1) [reply libs/shared/data/feedback/mocks/sample-reducer/index.ts:1]: Great question, I guess I had a reason... but know I don't know what was it 🤣

File: libs/shared/data/feedback/mocks/sample-reducer/selectors.ts (hunk 1 above)
Comment (krysia1) [reply libs/shared/data/feedback/mocks/sample-reducer/selectors.ts]: ok

File: libs/shared/data/features/sample-reducer/actions.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { sampleSlice } from "./sample-slice";
```
Comment (kacperChlon) [inline libs/shared/data/features/sample-reducer/actions.ts:1]: this all files shouldn't be in features directory, this is not 'features, it's features/sample/....

Please move all files related to sample-feature to features/sample dir

File: tsconfig.base.json [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "paths": {
       "@patronage-web/features-feedback": ["libs/features/feedback/src/index.ts"],
       "@patronage-web/shared": ["libs/shared/src/index.ts"],
+      "@patronage-web/shared-data": ["libs/shared/data/index.ts"],
```
[the remark this answers is not in the document]
Comment (kacperChlon) [reply tsconfig.base.json:26]: No, it means looks good to me ;)

File: libs/shared/data/features/sample-reducer/selectors.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-cycle
```
[the remark this answers is not in the document]
Comment (krysia1) [reply libs/shared/data/features/sample-reducer/selectors.ts:1]: ok

File: libs/shared/data/features/sample-reducer/actions.ts (hunk 3 above)
Comment (krysia1) [reply libs/shared/data/features/sample-reducer/actions.ts]: I see, ok done 😸

**Review by kacperChlon — APPROVED** [review]

**Pull request merged**